common/sc_popen.cpp

```diff
@@ -163,10 +163,11 @@ std::tuple<pid_t, FILE*> sc_popen_argv(const std::vector<std::string>& strings,
         strings.begin(), strings.end(), std::string(),
         [](const std::string& a, const std::string& b) -> std::string { return a + (a.length() > 0 ? " " : "") + b; });
 
-    return sc_popen(std::move(commandLine), type);
+    return sc_popen_c(std::move(commandLine).data(), type.data());
```

Comment:
Since you are just calling a member function, there's no point in moving the object!

Reply:
Thanks! I removed `move`.